src/main/java/com/tarento/assetManagement/dao/mapper/AuditMapper.java

```diff
@@ -0,0 +1,10 @@
+package com.tarento.assetManagement.dao.mapper;
+
+import org.mapstruct.Mapper;
+
+import com.tarento.assetManagement.dao.dto.Audit;
+
+@Mapper(componentModel = "spring")
+public interface AuditMapper {
+    Audit map(com.tarento.assetManagement.dao.model.Audit audit);
```

Comment:
is it not possible to use one interface and have multiple transform method.
Ex:
Asset map(com.tarento.assetManagement.dao.model.Asset asset);
Audit map(com.tarento.assetManagement.dao.model.Audit audit);

Reply:
Will try and check this Maznarul